src/screens/SUE/SueListScreen.tsx

```diff
@@ -53,6 +55,22 @@ export const SueListScreen = ({ navigation, route }: Props) => {
       })
   );
 
+  const { data: servicesData } = useQuery([QUERY_TYPES.SUE.SERVICES, {}], () =>
+    getQuery(QUERY_TYPES.SUE.SERVICES)({})
+  );
```

Comment:
we do not need the `{}`s, or am I wrong? is this something "for the future"?

Reply:
No, nothing for the future. I added an empty object because I don't need `queryVariables` for services. But it can be removed.